screens/MatchingScreen.tsx

```diff
@@ -53,11 +54,14 @@ export default function MatchingScreen({
     getFlirtsForCompany()
       .then(setFlirts)
       .catch(() => {});
-    console.log('flirts', flirts);
   }, [refresh]);
 
+  let uniqueFlirts = flirts.filter((flirt, index, self) => {
+    return self.findIndex(f => f.student.name === flirt.student.name) === index;
+  });
+  // Prevents from showing duplicate flirts
```

Comment:
Wait what, why are duplicate flirts allowed in the first place??? That sounds like a pretty significant bug in the backend...